src/accessor/index.js

```diff
@@ -11,6 +11,7 @@ module.exports = function() {
     rsi: require('./rsi'),
     trendline: require('./trendline'),
     value: require('./value'),
-    volume: require('./volume')
+    volume: require('./volume'),
+    tick: require('./tick')
```

Comment:
Yes, we need to come up with a better domain terminology.
The context is within the streaming ticks, but the overall tick seems to be very overloaded term in d3 charts.
Sleeping on the terminology for now.